wormhole/recv.go

```diff
@@ -247,8 +247,8 @@ func (c *Client) Receive(ctx context.Context, code string) (fr *IncomingMessage,
 type IncomingMessage struct {
 	Name              string
 	Type              TransferType
-	TransferBytes     int
-	UncompressedBytes int
+	TransferBytes     int64
+	UncompressedBytes int64
```

Comment:
This would be a breaking change if there are existing users depending on these fields. 

To avoid breaking users I think we should keep these fields as `int`, mark them as deprecated and add two new fields `TransferBytes64` and `UncompressedBytes64` as `int64`s. We'll want to continue populating both the old and new fields. We should document on 32bit binaries these fields will not have the correct values for large binaries.

Reply:
Yes you are correct, I had not considered that.

I will look into this and update the PR, I guess https://github.com/golang/go/wiki/Deprecated is the way to mark fields as deprecated?

Reply:
Yeah the way it is shown in that wiki page is good. `net/http` has some examples of deprecated fields too.